sdk/tables/azure-data-tables/README.md

```diff
@@ -0,0 +1,197 @@
+# Azure Tables client library for Go
+
+Azure Data Tables is a NoSQL data storage service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS.
+Tables scales as needed to support the amount of data inserted, and allows for the storing of data with non-complex accessing.
+The Azure Tables client can be used to access Azure Storage or Cosmos accounts.
```

Comment:
Why do we have this readme markdown duplicated twice, here and at the root of this project?

Reply:
both folders need readme, i don't know what else to put there